addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/NetatmoBindingConstants.java

```diff
@@ -105,10 +105,18 @@
     public static final String CHANNEL_SETPOINT_TEMP = "Sp_Temperature";
     public static final String CHANNEL_THERM_RELAY = "ThermRelayCmd";
     public static final String CHANNEL_THERM_ORIENTATION = "ThermOrientation";
+    public static final String CHANNEL_PROGRAM_NAME = "ProgramName";
     public static final String CHANNEL_CONNECTED_BOILER = "ConnectedBoiler";
     public static final String CHANNEL_LAST_PLUG_SEEN = "LastPlugSeen";
     public static final String CHANNEL_LAST_BILAN = "LastBilan";
 
+    public static final String CHANNEL_SETPOINT_MODE_MANUAL = "manual";
+    public static final String CHANNEL_SETPOINT_MODE_AWAY = "away";
+    public static final String CHANNEL_SETPOINT_MODE_HG = "hg";
```

Comment:
What is the meaning of `hg`?

Reply:
hg stands for "Hors Gel" in french. This mean minimum temperature avoiding water freeze.

Reply:
I see. We have six different modes for this channel. What about adding an options list in the chanel-type definition? If we do that we can translate them for a better understanding.